src/components/HistoryTables/MessageHistory/hooks/useMethodName.ts

```diff
@@ -18,6 +24,13 @@ export const useMethodName = (
     skip: !address
   })
 
+  const [abi] = useAbi(address?.robust)
+  if (isDelegatedAddress(address?.robust) && !!abi) {
+    return getFEVMMethodName(params, abi)
+  } else if (isDelegatedAddress(address?.robust)) {
+    return 'Unknown'
+  }
+
```

Comment:
Something in here errors now on the transaction history view but not the error detail view, not sure if the `abi` has something to do with this...

Reply:
Might also have to do with SSR